containers/docker-compose.yaml

```diff
@@ -152,8 +152,10 @@ services:
       - GDS_BFF_SENTRY_DSN
       - GDS_BFF_SENTRY_ENVIRONMENT
       - GDS_BFF_SENTRY_TRACK_PERFORMANCE
-      - GDS_BFF_AUTH0_ISSUER=dev-bu-hbv3o.us.auth0.com
+      - GDS_BFF_AUTH0_DOMAIN=dev-bu-hbv3o.us.auth0.com
       - GDS_BFF_AUTH0_AUDIENCE=https://bff.vaspdirectory.net
+      - GDS_BFF_AUTH0_CLIENT_ID
+      - GDS_BFF_AUTH0_CLIENT_SECRET
```

Comment:
@masskoder you will have to update your `.env` file with these values from auth0 after we merge this PR.